app/templates/components/explore/side-bar.hbs

```diff
@@ -1,8 +1,11 @@
-<div class="map item">
+<div class="map item {{if isMapVisible 'mobile hidden'}}">
```

Comment:
Same goes for here. When `isMapVisible` is true the map should be visible but u are using class - `hidden`